src/Vlingo.Common/Identity/IIdentityGenerator.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+using System;
+using System.Security.Cryptography;
+using Vlingo.UUID;
+
+namespace Vlingo.Common.Identity
+{
+    public interface IIdentityGenerator
```

Comment:
My personal preference is to keep 1 class/interface per file but I'm not suggesting it's not ok like that

Reply:
I prefer that too. Sometimes, I feel just too lazy for internal classes. 😛